draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1250,21 +1250,24 @@ user agent MUST process the cookie-av as follows.
 If the attribute-name case-insensitively matches the string "Domain", the user
 agent MUST process the cookie-av as follows.
 
-1.  If the attribute-value is empty, the behavior is undefined. However, the
-    user agent SHOULD ignore the cookie-av entirely.
+1.  If the attribute-value is empty:
 
-2.  If the first character of the attribute-value string is %x2E ("."):
+    1. Let cookie-domain be the canonicalized request-host.
```

Comment:
If what we want to do is match Firefox's (and presumably Safari's) behavior as documented [here](https://github.com/httpwg/http-extensions/issues/1332#issuecomment-933967006) we shouldn't be setting the cookie-domain, that'd make a cookie that isn't host-only. We should keep it empty.

See [Step 8 of Section 5.5](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-08#section-5.5).

> If the domain-attribute is non-empty:
...
**Otherwise**:
        1.  Set the cookie's host-only-flag to true.